eggplant_project/settings/base.py

```diff
@@ -15,6 +15,10 @@
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.messages import constants as messages_constants
 
+COOP_NAME = 'KBHFF'
```

Comment:
No, I rather like to think that the project will grow and these sort of decisions about design patterns are important while they're easy to make and implement :)